.idea/misc.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'd delete all `.idea` stuff from version control and (optionally) put `.idea` into gitignore

Reply:
Done ))